search-app/docs/api.md

```diff
@@ -1529,7 +1529,12 @@ When `has_granules` is set to "true" or "false", results will be restricted to c
 
 Collection results are sorted by ascending entry title by default when a search does not result in a score.
 
-If a keyword search is performed then the search results will be sorted by relevance (score descending), then by temporal overlap if one or more temporal ranges are provided with the tie breaker being the EMS community usage score (also descending). The usage score comes from EMS metrics which contain access counts of the collections by short name and version. The metrics are ingested into the CMR.
+If a keyword search is performed then the search results will be sorted by:
+
+  * Relevance Score (descending), binned to the nearest 0.2. For example a score of 0.75 and 0.85 will be considered equal for sorting purposes.
+  * Temporal Overlap (descending), if one or more temporal ranges are provided.
```

Comment:
Might be worth defining temporal overlap. I assume that means we take the temporal coverage being searched against and if a collection covers that entire time range, it is 100% coverage. For example if searching for Jan2010-Jan2016, a collection with coverage of Jan2010-2013 would be considered 50%.